WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -23,6 +23,27 @@
     end
   end
 
+  context "delegates" do
+    it "requires that delegates are delegates" do
```

Comment:
As this test is only testing a future comp, I went with "delegates for future comps must be delegates".